virttest/virsh.py

```diff
@@ -3606,19 +3607,25 @@ def secret_get_value(uuid, options=None, **dargs):
     return command(cmd, **dargs)
 
 
-def secret_set_value(uuid, base64, options=None, **dargs):
+def secret_set_value(uuid, password, encode=False, options=None, **dargs):
     """
     Set a secret value
 
     :param uuid: secret UUID
-    :param base64: base64-encoded secret value
+    :param password: secret value
+    :param encode: if FALSE, that means you've already provided a base64-encoded
+                   password. if TRUE, will base64-encode password before use.
```

Comment:
Boolean value is case sensitive, so let's have doc string accordingly, False/True instead of FALSE/TRUE